packages/databricks-vscode/src/language/EnvironmentDependenciesInstaller.ts

```diff
@@ -53,10 +59,25 @@ export class EnvironmentDependenciesInstaller implements Disposable {
         this.onDidInstallAttemptEmitter.fire();
     }
 
+    async getSuggestedVersion() {
+        if (this.connectionManager.serverless) {
```

Comment:
at some point we need to take the client version into consideration but this is good for now.